packages/web-frontend/jest.config.js

```diff
@@ -1,15 +1,17 @@
+const baseConfig = require('../../jest.config-js.json');
+
 module.exports = {
-  collectCoverageFrom: ['src/**/*.{js,jsx}'],
+  ...baseConfig,
+  rootDir: '.',
   setupFiles: ['<rootDir>/config/polyfills.js'],
   testMatch: ['<rootDir>/src/**/__tests__/**/*.js?(x)', '<rootDir>/src/**/?(*.)(spec|test).js?(x)'],
-  testEnvironment: 'node',
   testURL: 'http://localhost',
   transform: {
-    '^.+\\.(js|jsx)$': '../../node_modules/babel-jest',
+    ...baseConfig.transform,
     '^.+\\.css$': '<rootDir>/config/jest/cssTransform.js',
     '^(?!.*\\.(js|jsx|css|json)$)': '<rootDir>/config/jest/fileTransform.js',
   },
-  transformIgnorePatterns: ['[/\\\\]node_modules[/\\\\].+\\.(js|jsx)$'],
+  transformIgnorePatterns: [`/node_modules/(?!(@?react-leaflet))`],
```

Comment:
Some syntax in react-leaflet was causing the new jest version to fail, so allowed it to be transformed